src/azure-cli/azure/cli/command_modules/appservice/appservice_environment.py

```diff
@@ -50,7 +50,7 @@ def show_appserviceenvironment(cmd, name, resource_group_name=None):
     return ase_client.get(resource_group_name, name)
 
 
-def create_appserviceenvironment_arm(cmd, resource_group_name, name, subnet, kind='ASEv2',
+def create_appserviceenvironment_arm(cmd, resource_group_name, name, subnet, kind='ASEv3',
```

Comment:
May I ask if this change of default behavior will cause a breaking change? If so, please add the `BREAKING CHANGE` tag in the PR title